tests/filecheck/dialects/builtin/unrealized_conv_cast.mlir

```diff
@@ -1,16 +1,22 @@
-// RUN: xdsl-opt %s | xdsl-opt --print-op-generic | filecheck %s
+// RUN: xdsl-opt %s | xdsl-opt --print-op-generic | xdsl-opt | filecheck %s
 
 // CHECK: module
 "builtin.module"() ({
   "func.func"() ({
-    // CHECK: %0 = "arith.constant"() {"value" = 0 : i64} : () -> i64
+    // CHECK: %0 = arith.constant 0 : i64
     %0 = "arith.constant"() {"value" = 0 : i64} : () -> i64
-    // CHECK: %1 = "builtin.unrealized_conversion_cast"(%0) : (i64) -> f32
+    // CHECK: %1 = builtin.unrealized_conversion_cast %0 : i64 to f32
     %1 = "builtin.unrealized_conversion_cast"(%0) : (i64) -> f32
-    // CHECK: %2 = "builtin.unrealized_conversion_cast"(%0) : (i64) -> i32
+    // CHECK: %2 = builtin.unrealized_conversion_cast %0 : i64 to i32
     %2 = "builtin.unrealized_conversion_cast"(%0) : (i64) -> i32
-    // CHECK: %3 = "builtin.unrealized_conversion_cast"(%0) : (i64) -> i64
+    // CHECK: %3 = builtin.unrealized_conversion_cast %0 : i64 to i64
     %3 = "builtin.unrealized_conversion_cast"(%0) : (i64) -> i64
+    // CHECK: %4 = builtin.unrealized_conversion_cast to i64
```

Comment:
Oh, never seen that case, what is a cast from nothing to a type? 🤔
Looked for a case because I see it specially implemented in the parser.

Reply:
[UnrealizedConversionCastOp](https://mlir.llvm.org/docs/Dialects/Builtin/#builtinunrealized_conversion_cast-unrealizedconversioncastop) seems has 0-1 conversion.